src/advanced/components/CartItemBox.tsx

```diff
@@ -0,0 +1,45 @@
+import { CartItem as CartItemType, Product } from "../model/types";
+import { CartItem } from "./CartItem";
+
+interface Props {
+  onClickRemove: (item: Pick<CartItemType, "id" | "selectedQuantity">) => void;
+  increaseCartItemQuantity: (item: Pick<CartItemType, "id">) => void;
+  decreaseCartItemQuantity: (item: Pick<CartItemType, "id">) => void;
+  cartItems: CartItemType[];
+  productList: Product[];
+}
+
+export const CartItemBox = ({
+  onClickRemove,
+  increaseCartItemQuantity,
+  decreaseCartItemQuantity,
+  cartItems = [],
+  productList = [],
+}: Props) => {
+  return (
+    <div id="cart-items">
+      {cartItems.map((cartItem) => {
+        const product = productList.find((x) => x.id === cartItem.id);
```

Comment:
아래처럼 쓰면 if product null 조건문을 없앨 수 있어요
```suggestion
      {cartItems
        .filter((cartItem) => productList.some((product) => product.id === cartItem.id))
        .map((cartItem) => {
        const product = productList.find((x) => x.id === cartItem.id);
```